eng/illink-restore.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
+
+  <ItemGroup>
+    <PackageReference Include="illink.tasks" Version="$(ILLinkTasksPackageVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+    <ILLinkTfm Include="netcoreapp2.0" />
+    <ILLinkTfm Include="net46" />
+  </ItemGroup>
+
+  <Target Name="IncludeToolsFiles"
+          Inputs="%(ILLinkTfm.Identity)"
+          Outputs="$(ILLinkDir)"
```

Comment:
I don't think this is working correctly.  I believe inputs/outputs need to be files.  Since you use skipunchanged below I'd just omit inputs/outputs entirely.